configure.ac

```diff
@@ -1329,6 +1329,20 @@ AC_ARG_WITH(useragent, [AS_HELP_STRING([--with-useragent@<:@=AGENT@:>@], [User a
 
 # }}}
 
+# --with-max-data-len {{{
+AC_ARG_WITH(max-data-len, [AS_HELP_STRING([--with-max-data-len@<:@=VALUE@:>@], [Maximum length of data buffers])],
+[
+    if test "x$withval" != "x" && test $withval -gt 0
```

Comment:
I haven't checked, but how portable is test -gt?
I'm fine with leaving this check out. If a user decided to pass -1 to this, it's his own fault.